gradle.properties

```
@@ -1,8 +1,3 @@
 kotlin.code.style=official
-kotlin_version=2.0.0-RC3
-coroutines_version=1.8.0
-logback_version=1.3.14
-datetime_version=0.6.0-RC.2
-ktor_version=2.3.10
-retrofit_version=2.11.0
-serialization_version=1.6.3
\ No newline at end of file
+
+org.gradle.configuration-cache=true
```

WDYT about enabling Build Cache and Parallel too?

```suggestion
org.gradle.configuration-cache=true
org.gradle.caching=true
org.gradle.parallel=true
```